src/plugins/intel_gpu/src/graph/roi_align.cpp

```diff
@@ -14,16 +16,17 @@ primitive_type_id roi_align::type_id() {
     return &instance;
 }
 
-roi_align_inst::typed_primitive_inst(network& network, roi_align_node const& node)
-    : parent(network, node) {}
+roi_align_inst::typed_primitive_inst(network& network, roi_align_node const& node) : parent(network, node) {}
 
 layout roi_align_inst::calc_output_layout(roi_align_node const& node) {
     auto primitive = node.get_primitive();
     auto input_layout = node.input(0).get_output_layout();
     auto rois_layout = node.input(1).get_output_layout();
     auto num_rois = rois_layout.batch();
     auto num_channels = input_layout.feature();
-    return layout(input_layout.data_type, format::bfyx, {num_rois, num_channels, primitive->pooled_h, primitive->pooled_w});
+    return layout(input_layout.data_type,
+                  format::bfyx,
```

Comment:
if input layout and output are the same, could you use input here instead of direct value (to potentially support blocked layouts)